### ISSUE TITLE
Unable to reproduce paper performance

### ISSUE THREAD

**Issue by QiuHeqian** [yaoyao-liu/CL-DETR#5]
Thank you very much for the author's contribution. I tried to download and run the code for Google Cloud Drive, but the performance was very low and could not reproduce the performance of the paper. When will the author release the official code? Looking forward to your reply！

**Comment by Xuxiaoxiaohaha** [comment]
> Thank you very much for the author's contribution. I tried to download and run the code for Google Cloud Drive, but the performance was very low and could not reproduce the performance of the paper. When will the author release the official code? Looking forward to your reply！

It may have something to do with the random seed.You can try other seeds,which worked for me.

**Comment by yaoyao-liu** [comment]
Thank you so much for your interest in our work. It seems that the first phase training code has some issues. You may use the standard DETR code to train the first phase, and load the model for the following phases. I will get the code organized and release the official final version soon. If you have any further questions, please feel free to contact me.

**Comment by Xuxiaoxiaohaha** [comment]
> Thank you very much for the author's contribution. I tried to download and run the code for Google Cloud Drive, but the performance was very low and could not reproduce the performance of the paper. When will the author release the official code? Looking forward to your reply！

It may be caused by the pytorch version.I got poor (almost 0) results before with torch=1.8.1, cuda10.2, but got better results with torch=1.13.0, cuda11.7.

**Comment by yaoyao-liu** [comment]
@QiuHeqian 

Thanks for your interest in our work. I updated the project and uploaded checkpoints and log files. Please feel free to contact me if you still have any questions.

Best,

Yaoyao
